src/py42/modules/archive.py

```diff
@@ -106,3 +106,16 @@ def get_all_device_restore_history(self, days, device_id):
             that each contain a page of restore history.
         """
         return self._archive_client.get_all_restore_history(days, u"computerId", device_id)
+
+    def update_cold_storage_purge_date(self, archive_guid, purge_date):
+        """Updates the cold storage purge date for a specified archive.
+        REST Documentation <https://www.crashplan.com/apidocviewer/#ColdStorage-put>
```

Comment:
This doc link is a little off. It should all be surrounded in tick marks and should trail with two underscores, like this:

``REST Documentation <https://www.crashplan.com/apidocviewer/#ColdStorage-put>``__

Reply:
Also, for consistency's sake, can you link to `https://console.us.code42.com/apidocviewer/#ColdStorage-put` instead of crashplan?

Reply:
Updated. My habit is to use crashplan.com when looking at the documentation, so I didn't even think about it.